### DISCUSSION TITLE
Pagination performace with a small components

### DISCUSSION THREAD

**Discussion by antoniovuono** [gusgard/react-native-swiper-flatlist#133] [Q&A]
Hey guys, I'm trying to use a lib and I got a problem. 

When I try to use a swiper list with small components the pagination performance doesn't be fluid.

Example:
![Captura de Tela 2022-04-07 às 10 53 35](https://user-images.githubusercontent.com/7297243/162215409-1e815cd9-f7a9-498f-9f55-54e8e70e354e.png)

The render components size has almost 45% of the screen and the red balls do not change with fluidity.

```ts
 <SwiperFlatList
          style={{
            marginLeft: 10,
          }}
          showPagination
          onChangeIndex={onChangeIndex}
          renderAll
          paginationDefaultColor={Theme.colors.gray19}
          paginationActiveColor={Theme.colors.red}
          paginationStyle={{ width: '100%' }}
          paginationStyleItem={{
            marginTop: 50,
            width: 10,
            height: 10,
            marginRight: -3,
          }}
          horizontal
          showsHorizontalScrollIndicator={false}
          data={items}
          keyExtractor={item => (<Component />)}
      />

```

Any suggestion to make the pagination props more fluid? 

Thanks a lot.

**Comment by gusgard** [comment]
We released 3.2.2, with some improvements to avoid re-render it should improve performance. Let me know if it helped
